src/cryptoadvance/specter/devices/hwi/jade.py

```diff
@@ -671,9 +679,10 @@ def _get_device_entry(device_model: str, device_path: str) -> Dict[str, Any]:
 
         client = None
         with handle_errors(common_err_msgs["enumerate"], d_data):
-            client = JadeClient(device_path, password, expert, chain, timeout=1)
-            d_data["fingerprint"] = client.get_master_fingerprint().hex()
-
+            client = JadeClient(device_path, password, expert, chain, unlock, timeout=1)
+            # We don't get the fingerprint from a locked Jade
+            if client and unlock:
```

Comment:
Even if 'unlock=False' the jade might be unlocked - ie. if the jade was unlocked already - and you *could* get a fingerprint if you want one.  
I guess the you'd need to 'get_version_info()', get 'JADE_STATE' and test for 'READY' ?

Reply:
I've renamed `unlock` to `unlock_required` in ...

As of now, `unlock_required` is only `False` upon startup. Technically, yes, the Jade could have been unlocked by some other app, but the enumerate call on startup does not actually need the fingerprint. We could also just use `is_startup` as a param for the Jade client / the enumerate function but I thought it'd be better to add something more generic.  Should I change this? 

It also not the end of the world if try to get the fingerprint from the Jade if we can't unlock / auth, looks sth. like this then:

`[{'type': 'jade', 'model': 'jade', 'path': '/dev/cu.usbserial-56D40038341', 'needs_pin_sent': False, 'needs_passphrase_sent': False, 'error': 'Could not open client or get fingerprint information: Unhandled network: ', 'code': -7}]`